doc/firehol/firehol-mac.5.md

```diff
@@ -30,7 +30,7 @@ The `mac` helper command DROPs traffic from the *IP* address that was not
 sent using the *macaddr* specified.
 
 When packets are dropped, a log is produced with the label "MAC
-MISSMATCH" (sic.). `mac` obeys the default log limits (see
+MISMATCH" (sic.). `mac` obeys the default log limits (see
```

Comment:
Unfortunately I don't think we should take this particular fix, as if someone is matching this values in their logs we will break their setup.

The (sic.) indicates we know this is a mistake, so if we were to fix it, we'd want to remove that at the same time, but I don't think we can.

Reply:
Reverted.